src/main/kotlin/org/malachite/estella/services/SecurityService.kt

```diff
@@ -103,7 +103,18 @@ class SecurityService(
     fun checkUserRights(jwt: String?, userId: Int): Boolean =
         getUserFromJWT(jwt)?.let { it.id == userId } ?: false
 
+    fun checkJobSeekerRights(jwt: String?, id: Int) =
+        checkUserRights(jwt, id) && getJobSeekerFromJWT(jwt) != null
+
+    fun checkHrRights(jwt: String?, id: Int) =
+        checkUserRights(jwt, id) && getHrPartnerFromJWT(jwt) != null
+
+    private fun User.getUserType(): String =
+        jobSeekerRepository.findByUserId(this.id!!).orElse(null)?.let { "job_seeker" } ?:
```

Comment:
Czy nie przydałoby się ujednolicić że mamy w jobSeeker, hrPartner żeby wywoływać albo findByUserId albo findById

Reply:
Szczerze powiem, że nie wiem, jak na to Hibernate zareaguje.

Reply:
Znaczy wiesz u jobSeekera i hrPartnera id jest taki sam jak dla odpowiadającego usera, więc nie powinno

Reply:
W sensie nie wiem, jak zareaguje na zmianę nazwy metody XD